bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschSHCBridgeHandler.java

```diff
@@ -287,8 +287,17 @@ private void handleLongPollResult(LongPollResult result) {
     }
 
     private void handleLongPollFailure(Throwable e) {
-        logger.warn("Long polling failed", e);
-        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.COMMUNICATION_ERROR, "Long polling failed");
+        logger.warn("Long polling failed, will try to reconnect", e);
+        BoschHttpClient httpClient = this.httpClient;
+        if (httpClient == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.COMMUNICATION_ERROR,
+                    "Long polling failed and could not be restarted because http client is null");
```

Comment:
Can you use a ``@text/offline....newError"`` placeholder and add the text into ``boschshc.properties``. I think this error will appear more often and we should use i18n support.

Reply:
I agree, let's use the i18n for every error message from now on. This will make it possible for use to provide a German translation later on :) I can imagine that one of our users would help with translating the file once we have it filled.